### DISCUSSION TITLE
Ability to customize the password protected page

### DISCUSSION THREAD

**Discussion by vgstef** [elementor/elementor#23443] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I want to have a page with password protection. But there's no way to customize the password protected section/form.

Elementor should have the possibility to customize that page.

### Describe the solution you'd like

It could be the same kind of option in the Theme builder (just like error page 404). So we can assign our custom design to the password protection page type.

### Describe alternatives you've considered

I've looked at Addons, but it's too expensive for just this function for one page.

I could code in php/css, but that's not convenient for fast customization, translation, and client usage.

**Comment by ckcelite** [comment]
I would also like to know how to stylize password protected pages in WordPress or Elementor
